module/documents/activity/order.mjs

```diff
@@ -0,0 +1,278 @@
+import ActivityMixin from "./mixin.mjs";
+import OrderActivityData from "../../data/activity/order-data.mjs";
+import OrderUsageDialog from "../../applications/activity/order-usage-dialog.mjs";
+
+/**
+ * @typedef {ActivityUseConfiguration} OrderUseConfiguration
+ * @property {object} [costs]
+ * @property {number} [costs.days]               The cost of executing the order, in days.
+ * @property {number} [costs.gold]               The cost of executing the order, in gold.
+ * @property {boolean} [costs.paid]              Whether the gold cost has been paid.
+ * @property {object} [craft]
+ * @property {string} [craft.item]               The item being crafted or harvested.
+ * @property {number} [craft.quantity]           The quantity of items to harvest.
+ * @property {object} [trade]
+ * @property {boolean} [trade.sell]              Whether the trade was a sell operation.
+ * @property {object} [trade.stock]
+ * @property {boolean} [trade.stock.stocked]     Whether the order was to fully stock the inventory.
+ * @property {boolean} [trade.stock.value]       The base value of goods transacted.
+ * @property {object} [trade.creatures]
+ * @property {string[]} [trade.creatures.buy]    Additional animals purchased.
+ * @property {boolean[]} [trade.creatures.sell]  Whether a creature in a given slot was sold.
+ * @property {number} [trade.creatures.price]    The base value of the animals sold.
+ */
+
+/**
+ * An activity for issuing an order to a facility.
+ */
+export default class OrderActivity extends ActivityMixin(OrderActivityData) {
+  /* -------------------------------------------- */
+  /*  Model Configuration                         */
+  /* -------------------------------------------- */
+
+  /** @inheritDoc */
+  static metadata = Object.freeze(foundry.utils.mergeObject(super.metadata, {
+    configurable: false,
```

Comment:
Does this make more sense here or in `CONFIG.activityTypes`. Right now we have `hidden` in the configuration object which controls whether activities are displayed in the creation list, so perhaps this makes sense alongside it?